### PR TITLE
Implement dateValidator and datetimeValidator fillers

### PR DESCRIPTION
**Pull request by alkallio**

### CODE REVIEW

**Comment by alkallio** [comment]
Should we drop testing in php 7.0, 7.1, nightly and add 7.2, 7.3?

**Comment by akalineskou** [comment]
dont forget to push the changes

File: src/DateValidatorFiller.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+
```
Comment (nohponex) [inline src/DateValidatorFiller.php:3]: declare strict

File: tests/src/DateValidatorFillerTest.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?php
+
```
Comment (nohponex) [inline tests/src/DateValidatorFillerTest.php:3]: declare strict everywhere

**Review by nohponex — CHANGES_REQUESTED** [review]
declare strict everywhere

**Comment by alkallio** [comment]
Squash when merge.

**Review by nohponex — APPROVED** [review]

**Pull request merged**